packages/lab/src/stepper-input/useStepperInput.ts

```diff
@@ -66,59 +85,43 @@ export const useStepperInput = (
 
   const isAtMax = () => {
     if (currentValue === undefined) return true;
-    return toFloat(currentValue) >= max || (max === 0 && currentValue === "");
+    return toFloat(currentValue) >= max;
   };
 
   const isAtMin = () => {
     if (currentValue === undefined) return true;
-    return toFloat(currentValue) <= min || (min === 0 && currentValue === "");
+    return toFloat(currentValue) <= min;
   };
 
-  const decrement = () => {
+  const decrement = (event?: SyntheticEvent) => {
     if (currentValue === undefined || isAtMin()) return;
-    let nextValue = currentValue === "" ? -step : toFloat(currentValue) - step;
-
-    // Set value to `max` if it's currently out of range
-    if (max !== undefined && isOutOfRange()) nextValue = max;
-
-    setNextValue(nextValue);
+    let nextValue = toFloat(currentValue) - step;
+    if (nextValue < min) nextValue = min;
```

Comment:
Is this covered in test?

Reply:
added:
"does not decrement below the minimum value"
and
"does not increment above the maximum value"